shared/chat/conversation/attachment-fullscreen/index.desktop.tsx

```diff
@@ -117,7 +115,11 @@ class _Fullscreen extends React.Component<Props & Kb.OverlayParentProps, State>
                   {!this.props.isVideo ? (
                     <Kb.OrientedImage
                       src={this.props.path}
-                      style={this.state.isZoomed ? styles.imageZoom : styles.imageFit}
+                      style={
+                        this.state.isZoomed
+                          ? (styles.imageZoom as Styles.StylesCrossPlatform)
```

Comment:
there are several examples of us casting or doing `as const` to help newer typescript not get bogged down in overly complex types